src/sltyledComponents/pokeTabs.js

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import styled from 'styled-components';
+import { changeTab } from '../actions/pokemonActions';
+
+const tabs = [
+    {
+        title: 'Stats',
+        index: 0
+    },
+    {
+        title: 'Images',
+        index: 1
+    }
+];
+
+
+const Tab = styled.div`
+flex: 1;
+cursor: pointer;
+padding: 10px 0 ;
+font-weight: ${({active}) => active ? 'bold' : ""};;
+background-color: ${({active, color}) => active ? color : "lightgray"};
+color: ${({active}) => active ? 'white' : "black"};
+margin: 0 5px;
+border-radius: 5px;
+`;
+
+const Tabs = styled.div`
+display: flex;
+`;
+
+const TabsContainer = styled.div`
+flex: 1;
+`;
+
+
+const Sprite = styled.img`
+min-width: 200px;
+box-shadow: 0 2px 5px 0 rgba(0,0,0,0.16), 0 2px 10px 0 rgba(0,0,0,0.12);
+margin: 10px;
+background-color: white;
+`;
+
+const ImagesContainer = styled.div`
+display: flex;
+width: 100%;
+height: 100%;
+justify-content: center;
+flex-wrap: wrap;
+`;
+
+const Stat = styled.div`
+position: relative;
+padding: 10px 60px;
+font-size: 1.1rem;
+font-weight: bold;
+&::before {
```

Comment:
Many pseudo-elements make DOM tree harder to read. We should use norma elements instead.